src/fosslight_prechecker/_download_lic.py

```
@@ -83,7 +84,20 @@ def present_license_file(path_to_check: str, lic: str) -> bool:
-    reuse_return_code = reuse_download(parsed_args, prj)
+    # suppress direct prints by passing a devnull writer to reuse, and python-level redirection
+    reuse_return_code = None
+    try:
+        with open(os.devnull, 'w') as devnull_py:
+            with redirect_stdout(devnull_py), redirect_stderr(devnull_py):
+                try:
+                    reuse_return_code = reuse_download(parsed_args, prj, out=devnull_py)
+                except TypeError:
+                    try:
+                        reuse_return_code = reuse_download(parsed_args, prj, devnull_py)
+                    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)